comp/appoint-tech-day-safeguarding-officers.yaml

```diff
@@ -1,6 +1,6 @@
-summary: Appoint safeguarding officers
+summary: Appoint safeguarding officers for each tech day
```

Comment:
What's the expectation in terms of when this task will be "done"?
Tasks which relate to several things but which don't have an easy breakdown into subtasks are (as we've seen) hard to keep track of since they can spend a long time in a semi-complete state where nothing is progressing.

Given that assigning safeguarding officers is just part of the process for tech days (I realise that's not written up, but it should be https://github.com/srobo/runbook/issues/139) I'm not sure that this being a separate task is especially useful.

Reply:
We've now consolidated tech day planning into a single task where certain steps will need to be done for each tech day. When we know how many tech days a year will have we can always duplicate the task for each tech day.